r/pkgdown/assets/versions.html

```diff
@@ -1,9 +1,11 @@
 <!DOCTYPE html>
 <html>
-<body><p><a href="../dev/r/">13.0.0.9000 (dev)</a></p>
-<p><a href="../r/">13.0.0.1 (release)</a></p>
+<body><p><a href="../dev/r/">14.0.0.9000 (dev)</a></p>
+<p><a href="../r/">14.0.0 (release)</a></p>
+<p><a href="../13.0/r/">13.0.0.1</a></p>
 <p><a href="../12.0/r/">12.0.1.1</a></p>
-<p><a href="../11.0/r/">11.0.0.3</a></p>
+<p><a href="../12.0/r/">12.0.0</a></p>
```

Comment:
Should we remove this?

Reply:
This is part of the automated diff generated when `post-11-bump-versions.sh` is run. I also found that strange but might mean there is something wrong when bumping the versions.
@thisisnic @paleolimbot @assignUser is there anything we should fix when automatically bumping versions on the `versions.html` file?

Reply:
I'm not familiar with the doc setup so :shrug: sorry^^